integration/test-integration/src/test/java/org/wso2/choreo/connect/tests/testcases/withapim/apikey/APIKeyTestCase.java

```diff
@@ -1,241 +0,0 @@
-    private static final String APP_NAME = "APIKeyTestApp";
-    private String apiKey;
-    private String apiKeyForIPTest;
-    private String apiKeyForRefererTest;
-    private String tamperedAPIKey;
-    private String applicationId;
-    private String apiId;
-    private String endPoint;
-    private String jwtEndpoint;
-
-    @BeforeClass(description = "Initialise the setup for API key tests")
-    void start() throws Exception {
-        super.initWithSuperTenant();
-
-        String targetDir = Utils.getTargetDirPath();
-        String filePath = targetDir + ApictlUtils.OPENAPIS_PATH + "api_key_openAPI.yaml";
-
-        // enable apikey in apim
-        JSONArray securityScheme = new JSONArray();
-        securityScheme.put("oauth_basic_auth_api_key_mandatory");
-        securityScheme.put("api_key");
-
-        JSONObject apiProperties = new JSONObject();
-        apiProperties.put("name", SAMPLE_API_NAME);
-        apiProperties.put("context", "/" + SAMPLE_API_CONTEXT);
-        apiProperties.put("version", SAMPLE_API_VERSION);
-        apiProperties.put("provider", user.getUserName());
-        apiProperties.put("securityScheme", securityScheme);
-        apiId = PublisherUtils.createAPIUsingOAS(apiProperties, filePath, publisherRestClient);
-
-        publisherRestClient.changeAPILifeCycleStatus(apiId, "Publish");
-
-        // creating the application
-        Application app = new Application(APP_NAME, TestConstant.APPLICATION_TIER.UNLIMITED);
-        applicationId = StoreUtils.createApplication(app, storeRestClient);
-
-        PublisherUtils.createAPIRevisionAndDeploy(apiId, publisherRestClient);
-
-        StoreUtils.subscribeToAPI(apiId, applicationId, TestConstant.SUBSCRIPTION_TIER.UNLIMITED, storeRestClient);
-
-        endPoint = Utils.getServiceURLHttps(SAMPLE_API_CONTEXT + "/1.0.0/pet/1");
-        jwtEndpoint = Utils.getServiceURLHttps(SAMPLE_API_CONTEXT + "/1.0.0/jwtheader");
-
-        // Obtain API keys
-        APIKeyDTO apiKeyDTO = StoreUtils.generateAPIKey(applicationId, TestConstant.KEY_TYPE_PRODUCTION,
-                storeRestClient);
-        apiKey = apiKeyDTO.getApikey();
-        tamperedAPIKey = apiKey.substring(0, apiKey.length() - 400);
-
-        APIKeyDTO ipTestAPIKeyDTO = storeRestClient.generateAPIKeys(applicationId, TestConstant.KEY_TYPE_PRODUCTION,
-                -1, "192.168.1.1", null);
-        apiKeyForIPTest = ipTestAPIKeyDTO.getApikey();
-
-        APIKeyDTO refererTestAPIKeyDTO = storeRestClient.generateAPIKeys(applicationId, TestConstant.KEY_TYPE_PRODUCTION,
-                -1, null, "http://www.abc.com");
-        apiKeyForRefererTest = refererTestAPIKeyDTO.getApikey();
-
-        Utils.delay(TestConstant.DEPLOYMENT_WAIT_TIME, "Could not wait till initial setup completion.");
-    }
-
-    //    Invokes with tampered API key and this will fail.
-    @Test(description = "Test to detect wrong API keys")
-    public void invokeWithTamperedAPIKey() throws Exception {
```

Comment:
Here we are supposed to only remove this segment and rest is still valid. (Internal-Key)